.circleci/config.yml_disabled

```diff
@@ -1,3 +1,5 @@
+REMOVE ME
```

Comment:
Why not just `git rm` ?

Reply:
I am keeping it for reference. I still have to move docker-compose section to Github actions